### PR TITLE
[task] added some documentation for JepScrapper

### PR DESCRIPTION
**Pull request by lmilunovic**
Also tiny refactoring of setJeps method.

### CODE REVIEW

File: src/main/java/com/jvmrally/lambda/tasks/JepScraper.java [hunk 1]
```diff
@@ -58,7 +60,15 @@ public void run() {
         }
     }
 
+    /**
+     * Searches trough list of existing jeps list for news and changes from scraped jeps list.
```
Comment (Toby-Larone) [inline src/main/java/com/jvmrally/lambda/tasks/JepScraper.java]: Typo, `trough` -> `through

File: src/main/java/com/jvmrally/lambda/tasks/JepScraper.java [hunk 2]
```diff
@@ -149,6 +196,12 @@ private void updateJep(Jep existingJep, Jep jep) {
         sendUpdatedEmbed(existingJep, jep);
     }
 
+    /**
+     * Uses Jsoup to connect to JEP_URL in order to scrap html into com.jvmrally.lambda.jdk.Jep objects.
```
Comment (Toby-Larone) [inline src/main/java/com/jvmrally/lambda/tasks/JepScraper.java]: Typo `scrap` -> `scrape`

File: src/main/java/com/jvmrally/lambda/tasks/JepScraper.java (hunk 1 above)
Comment (lmilunovic) [reply src/main/java/com/jvmrally/lambda/tasks/JepScraper.java]: Fixed in next revison

File: src/main/java/com/jvmrally/lambda/tasks/JepScraper.java (hunk 2 above)
Comment (lmilunovic) [reply src/main/java/com/jvmrally/lambda/tasks/JepScraper.java]: Fixed in next revison

**Pull request merged**